Xamarin.Forms.Core/IndicatorView.cs

```diff
@@ -0,0 +1,256 @@
+	{
+		public static readonly BindableProperty PositionProperty = BindableProperty.Create(nameof(Position), typeof(int), typeof(IndicatorView), default(int), BindingMode.TwoWay, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty CountProperty = BindableProperty.Create(nameof(Count), typeof(int), typeof(IndicatorView), default(int), propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorCount((int)oldValue));
+
+		public static readonly BindableProperty MaximumVisibleCountProperty = BindableProperty.Create(nameof(MaximumVisibleCount), typeof(int), typeof(IndicatorView), int.MaxValue, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty IndicatorTemplateProperty = BindableProperty.Create(nameof(IndicatorTemplate), typeof(DataTemplate), typeof(IndicatorView), default(DataTemplate), propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty HidesForSingleIndicatorProperty = BindableProperty.Create(nameof(HidesForSingleIndicator), typeof(bool), typeof(IndicatorView), true, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty IndicatorColorProperty = BindableProperty.Create(nameof(IndicatorColor), typeof(Color), typeof(IndicatorView), Color.Default, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty SelectedIndicatorColorProperty = BindableProperty.Create(nameof(SelectedIndicatorColor), typeof(Color), typeof(IndicatorView), Color.Default, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty IndicatorSizeProperty = BindableProperty.Create(nameof(IndicatorSize), typeof(double), typeof(IndicatorView), -1.0, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty ItemsSourceProperty = BindableProperty.Create(nameof(ItemsSource), typeof(IEnumerable), typeof(IndicatorView), null, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetItemsSource((IEnumerable)oldValue));
+
+		public static readonly BindableProperty IndicatorLayoutProperty = BindableProperty.Create(nameof(IndicatorLayout), typeof(Layout<View>), typeof(IndicatorView), null, propertyChanged: TemplateUtilities.OnContentChanged);
+
+		public IndicatorView()
+			=> IndicatorLayout = new StackLayout { Orientation = StackOrientation.Horizontal };
+
+		IList<View> Items => IndicatorLayout.Children;
+
+		public Layout<View> IndicatorLayout
```

Comment:
What is the purpose of `IndicatorLayout` on iOS now that you're using a `UIPageControl`? Is this `StackLayout` being used for anything? How is it being used for Android?

Reply:
@adrianknight89 on iOS it is not used. But I really don't know how to manage such cases.

1) iOS uses a renderer
2) All other platforms use a cross-platform solution


For Android and all other platforms, IndicatorLayout is used as "layout strategy". By default, it's horizontal stack, but you can easily change it to Vertical stack or any other layout

Reply:
That's what I thought was going on. To be honest, people will be confused about this or potentially other properties having no effect on iOS due to the renderer being present. For Android and others, the path forward could be creating ViewRenderers (with shapes, pagination, etc.) from scratch. This requires more work, but it will solve this mismatch issue. In my opinion, if we are leveraging Core, then we should do it for all platforms, but I know you were asked to use a renderer. :)

Reply:
We are thinking in using Visual="Forms" for a cross platform solution , while No visual would use a native renderer , what do guys think of that approach?

Reply:
sounds good